build-ps/percona-server.spec

```diff
@@ -550,6 +550,18 @@ rm -rf %{buildroot}%{_bindir}/mysql_embedded
   rm -r $(readlink var) var
 %endif
 
+%pretrans -n Percona-Server-server%{product_suffix}
+if [ -d %{_datadir}/mysql ]; then
+  MYCNF_PACKAGE=$(rpm -qi `rpm -qf /usr/share/mysql` | grep -m 1 Name | awk '{print $3}')
+fi
+
+if [ "$MYCNF_PACKAGE" == "mariadb-libs" -o "$MYCNF_PACKAGE" == "mysql-libs" ]; then
+  MODIFIED=$(rpm -Va "$MYCNF_PACKAGE" | grep '/usr/share/mysql' | awk '{print $1}' | grep -c 5)
+  if [ "$MODIFIED" == 1 ]; then
```

Comment:
it is better to add check that '/usr/share/mysql' is not link

Reply:
Gotcha, will do